plugins/linters/reek/plugin.toml

```
@@ -0,0 +1,20 @@
+config_version = "0"
+
+[plugins.definitions.reek]
+runtime = "ruby"
+package = "reek"
+file_types = ["ruby"]
+latest_version = "6.3.0"
+known_good_version = "6.3.0"
+version_command = "reek --version"
+config_files = [".reek.yml"]
+description = "Ruby linter"
```

Using the [official description](https://github.com/troessner/reek?tab=readme-ov-file#code-smell-detector-for-ruby) is preferred when possible:

```suggestion
description = "Code smell detector for Ruby"
```